opcua/redis/streams.go

```diff
@@ -189,39 +176,23 @@ func decodeCreateChannel(event map[string]interface{}) (createChannelEvent, erro
 		id: read(event, "id", ""),
 	}
 
-	val, ok := metadata[protocol]
+	m, ok := metadata[keyProtocol]
 	if !ok {
 		return createChannelEvent{}, errMetadataType
 	}
-	if val.Namespace == "" {
-		return createChannelEvent{}, errMetadataAppID
-	}
 
-	cce.metadata = val
-	return cce, nil
-}
-
-func decodeUpdateChannel(event map[string]interface{}) (updateChannelEvent, error) {
-	strmeta := read(event, "metadata", "{}")
-	var metadata map[string]channelMetadata
-	if err := json.Unmarshal([]byte(strmeta), &metadata); err != nil {
-		return updateChannelEvent{}, err
-	}
-
-	uce := updateChannelEvent{
-		id: read(event, "id", ""),
+	om, ok := m.(map[string]interface{})
```

Comment:
What does `om` stand for? Without going into the usbject, var name is a bit confusing.